£Lerror_reason¢DtypeEvalueYFHTTPConnectionPool(host='closeup.ics.uci.edu', port=80): Max retries exceeded with url: /cgi-bin/aoife/plot_maize_all?data=%60lineup%60.%60pio99_fast2&ref_chr=1 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x0000000009B99208>, 'Connection to closeup.ics.uci.edu timed out. (connect timeout=2)'))Ihttp_code¢DtypeEvalueÛQdownload_complete¢DtypeEvalueı